<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServerControlView</class>
 <widget class="QWidget" name="ServerControlView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::FieldsStayAtSizeHint</enum>
     </property>
     <item row="0" column="0">
      <widget class="QLabel" name="portLabel">
       <property name="text">
        <string>Port</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="portSpinBox">
       <property name="minimum">
        <number>1024</number>
       </property>
       <property name="maximum">
        <number>9999</number>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="stopButton">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Stop</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="1">
      <layout class="QVBoxLayout" name="fileSelect">
       <item>
        <widget class="QLabel" name="fileLabel">
         <property name="text">
          <string>File</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="fileSelect2">
         <item>
          <widget class="QPushButton" name="fileSelectButton">
           <property name="text">
            <string>Select File</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="fileName">
           <property name="text">
            <string>No File Selected</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Server log</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTextBrowser" name="logBrowser"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
